1001-2000/LIT1824Mafteh.xml

```diff
@@ -78,15 +79,6 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
             <relation name="ecrm:CLP46i_may_form_part_of" active="LIT5139MagicPrayer" passive="LIT1824Mafteh"></relation>
```

Comment:
`active `must always be the current record 

so relation should be `saws:contains` 

(or you can put the relation as it is now under `LIT5139MagicPrayer`)

Reply:
(and added to LIT5139MagicPrayer)

that is 
either under LIT5139MagicPrayer we have the relation 
`<relation name="ecrm:CLP46i_may_form_part_of" active="LIT5139MagicPrayer" passive="LIT1824Mafteh"></relation>
`

or here in the LIT1824Mafteh record we have
` <relation name="saws:contains" active="LIT1824Mafteh" passive="LIT5139MagicPrayer"></relation>`

the current record is always the `active `member